standards/code-style.adoc

```diff
@@ -551,14 +557,13 @@ __match the component's lifecycle__. For example, if you have an Activity that
 implements `onCreate()`, `onDestroy()`, `onPause()` and `onResume()`, then the
 correct order is:
 
-[source, java]
+[source, kotlin]
 ----
-public class MainActivity extends Activity {
+class MainActivity : Activity() {
     //Order matches Activity lifecycle
-    @Override public void onCreate() { }
-    @Override public void onResume() { }
-    @Override public void onPause() { }
-    @Override public void onDestroy() { }
+    public override fun onResume() {}
```

Comment:
`onCreate()` is missing from the example, but mentioned above in the text.

Reply:
Updated.